src/components/ImageUploader/src/ImageSelection.vue

```diff
@@ -145,7 +140,7 @@ export default {
 }
 
 .ImageSelectionContainerError {
-	border: 2px solid var(--color-error);
+	border: 1px solid var(--color-error);
```

Comment:
i found this pretty hard to see on the styleguide, was this a design requested change?

Reply:
once you have checked with design please update the PR and/or resolve this comment thread

Reply:
Got confirmation, we're g2g